doc/atcommands.md

```diff
@@ -93,112 +95,114 @@ Output Example:
 
 ---------------------------------------
 
-@noask
+`@noask`
 
 Toggles automatic rejection of deals and invites.
 
 ---------------------------------------
 
-@noks
+`@noks`
 
 Toggles Kill Steal Protection.
 
 ---------------------------------------
 
-@font <0-9>
+`@font <0-9>`
 
 Sets client font (0 is the default).
 
 ---------------------------------------
 
-@agitstart
-@agitend
+`@agitstart
+@agitend`
 
 Starts or ends War of Emperium [FE] by invoking scripts with the
 OnAgitStart/OnAgitEnd labels.
 
 ---------------------------------------
 
-@agitstart2
-@agitend2
+`@agitstart2
```

Comment:
Same as for `@refresh`, it should be code block (triple back-tick)

Reply:
fixed.